recipes/yacht/post-link.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Installing pyo3-branchwater via pip..."
+"${PREFIX}/bin/pip" install pyo3-branchwater==0.8.1
```

Comment:
We can not do that ;)

Please create a recipe for that tool as well at https://github.com/conda-forge/staged-recipes/pulls